src/WingetCreateCore/Common/GitHub.cs

```diff
@@ -483,8 +486,23 @@ private async Task UpdateForkedRepoWithUpstreamCommits(Repository forkedRepo)
                     throw new NonFastForwardException(commitsAheadBy);
                 }
 
-                var upstreamBranchReference = await this.github.Git.Reference.Get(upstream.Id, $"heads/{upstream.DefaultBranch}");
-                await this.github.Git.Reference.Update(forkedRepo.Id, $"heads/{forkedRepo.DefaultBranch}", new ReferenceUpdate(upstreamBranchReference.Object.Sha));
+                // Octokit .NET doesn't support sync fork endpoint, so we make a direct call to the GitHub API.

```

Comment:
Great find! Thank you. What is an appropriate github/doc link to add to this comment here so we can continue leveraging the Github API (Octokit .net) once this issue is resolved? Or that change was by design?

Reply:
I created an issue myself for this request - which I can add a link to in the comment.

- https://github.com/octokit/octokit.net/issues/2989

Reply:
Awesome! I appreciate it. Adding the comment will help us remember to change this in the future 😊